pages/api/discord-thread.ts

```diff
@@ -94,22 +99,25 @@ async function fetchDiscordThread(
 ): Promise<VoteDiscussionT> {
   // First, fetch all messages in the evidence rational channel.
   const threadMsg = await getDiscordMessages(evidenceRationalDiscordChannelId);
+
   // Then, extract the timestamp from each message and for each timestamp relate
   // it to the associated threadId.
-  const timeToThread: { [key: string]: string } = {};
+  const timeToThread: { [key: string]: string | undefined } = {};
   threadMsg.forEach((message) => {
-    const time = extractValidateTimestamp(message.thread.name);
-    if (time) timeToThread[time.toString()] = message.thread.id;
+    const time = extractValidateTimestamp(message.content);
```

Comment:
this was causing the crash, when `name` was undefined

Reply:
maybe we need to do message.thread.name ?? message.content